CHANGES

```diff
@@ -1,4 +1,7 @@
-Upcoming release (1.0.0)
+Upcoming release
+================
+
```

Comment:
Maybe let's not add an "Upcoming release" entry until after the release. Just seems weird to have it in the official release docs, even if it'll show up in "latest" on the next commit.